### PR TITLE
🛥️⛵🚤🌊AU-CP Main

### PR DESCRIPTION
**Pull request by Gooochy**
**What kind of update does this PR provide?** *Please check*

- [x] new translations / updated translations / translation fixes
- [x] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**Which language(s) did you add/update translations for?**
*use the xx_XX notation for exact language!*
* en_AU

**What is included in your update?**

* AU's SES & Water Rescue CP

**Additional notes**
Please add any further notes here!
Heavy WIP

### CODE REVIEW

File: src/i18n/en_AU.ts [hunk 1]
```diff
@@ -710,7 +771,27 @@ export default {
                 staffList: 'Intensive Care Paramedic',
             },
         ],
-    },
+        'SES & Water Rescue': [
+            {
+               caption: 'Ocean Navigation',
+               duartion: '5 Days',
```
Comment (Suchty112) [inline src/i18n/en_AU.ts:777]: duration

File: src/i18n/en_AU.ts [hunk 2]
```diff
@@ -710,7 +771,27 @@ export default {
...
+               staffList: 'Ocean Navigation',
+            },
+            {
+               caption: 'SES Rescue',
+               duartion: '5 Days',
```
Comment (Suchty112) [inline src/i18n/en_AU.ts:782]: duration

File: src/i18n/en_AU.ts [hunk 3]
```diff
@@ -710,7 +771,27 @@ export default {
...
+               staffList: 'SES Rescue',
+            },
+            {
+               caption: 'Boat Captain Training',
+               duartion: '5 Days',
```
Comment (Suchty112) [inline src/i18n/en_AU.ts:787]: duration

File: src/i18n/en_US.ts [hunk 4]
```diff
@@ -1170,9 +1170,9 @@ export default {
         'Firefighters': {
             vehicles: {
                 'Fire trucks': [0, 1, 13, 18],
-                'Water Tanker': [7, 33],
+                'Water Tanker': [7, 33, 77, 78],
                 'Special vehicles': [2, 3, 4, 6, 8, 9, 12, 62, 63, 64, 65],
-                'Wildland vehicles': [30, 31, 32, 34, 38, 39, 40, 41, 77, 78],
+                'Wildland vehicles': [30, 31, 32, 34, 38, 39, 40, 41, 74,],
```
Comment (Suchty112) [inline src/i18n/en_US.ts:1175]: remove the ',' after 74

**Review by Suchty112 — CHANGES_REQUESTED** [review]

File: src/i18n/en_AU.ts [hunk 5]
```diff
@@ -710,7 +771,27 @@ export default {
...
+               staffList: 'Boat Captain Training',
+            },     
+            {
+              caption: 'Mobile Command Training'
```
Comment (Suchty112) [inline src/i18n/en_AU.ts:791]: and here is missing a comma

File: src/i18n/en_AU.ts [hunk 6]
```diff
@@ -710,7 +771,27 @@ export default {
...
+              duration: '5 Days',
+              staffList: 'Mobile Command Training
```
Comment (Suchty112) [inline src/i18n/en_AU.ts:793]: and here is missing the ' and a comma

Comment (Suchty112) [reply src/i18n/en_AU.ts:793]: this is not resolved actually ^^

Comment (Gooochy) [reply src/i18n/en_AU.ts]: Now it is :P

**Comment by Gooochy** [comment]
Sanni do you know how the new boat is written in red text?

File: src/i18n/en_AU.ts [hunk 7]
```diff
@@ -710,7 +771,28 @@ export default {
...
+               caption: 'Boat Captain Training',
+               duration: '5 Days',
+               staffList: 'Boat Captain Training',
+            },     
+            {
+              caption: 'Mobile Command Training',
+              duration: '5 Days',
+              staffList: 'Mobile Command Training',
+         },
+        ],
```
Comment (jxn-30) [inline src/i18n/en_AU.ts:795]:
```suggestion
       ],
    },
```

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/i18n/en_AU.ts [hunk 8]
```diff
@@ -566,7 +566,72 @@ export default {
                 },
             },
         },
-    },
+                },
```
Comment (jxn-30) [inline src/i18n/en_AU.ts:569]:
```suggestion
```

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/i18n/en_AU.ts (hunk 8 above)
Comment (jxn-30) [inline src/i18n/en_AU.ts:569]:
```suggestion
```

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Comment by Gooochy** [comment]
Jan do you know how exactly the VMR boat is written?

**Comment by jxn-30** [comment]
no.

**Comment by Suchty112** [comment]
> Sanni do you know how the new boat is written in red text?

SES Storm Trailers,  MVR Quads, MVR Boats plural
SES Storm Trailer, MVR Quad, MVR Boat singular

**Comment by Gooochy** [comment]
> > Sanni do you know how the new boat is written in red text?
> 
> SES Storm Trailers, MVR Quads, MVR Boats plural SES Storm Trailer, MVR Quad, MVR Boat singular

asfaik  MVR Quad and Storm Trailers arent required for missions

**Comment by Suchty112** [comment]
> > > Sanni do you know how the new boat is written in red text?
> > 
> > 
> > SES Storm Trailers, MVR Quads, MVR Boats plural SES Storm Trailer, MVR Quad, MVR Boat singular
> 
> asfaik MVR Quad and Storm Trailers arent required for missions

but you can use them in custom missions. That's how I know. And it can still be added. But please do it right ^^

**Comment by Suchty112** [comment]
I take it back. The trailers are requested. quad as well. but the quad is not displayed in the einsaetze.json.
https://www.missionchief-australia.com/einsaetze/534

**Comment by Suchty112** [comment]
> I take it back. The trailers are requested. quad as well. but the quad is not displayed in the einsaetze.json. https://www.missionchief-australia.com/einsaetze/534

coastal_rescue is the quad. and road_rescue is the trailer

File: src/modules/extendedCallWindow/i18n/en_AU.json [hunk 9]
```diff
@@ -154,14 +154,28 @@
...
+                     "MVR Quad",
+                     "MVR Quads"
+            ],
+            "vehicles": [38]
+            },
+                        {
                 "texts": [
                      "MVR Boat",
-                     "MVR Boats"
+                     "MVR Boat"
```
Comment (Suchty112) [inline src/modules/extendedCallWindow/i18n/en_AU.json:168]: why remove the s? plural is Boats :D

Comment (Gooochy) [reply src/modules/extendedCallWindow/i18n/en_AU.json:168]: Mistake will be fixed now

**Comment by Gooochy** [comment]
I think its done

**Comment by Suchty112** [comment]
> I think its done

nop. the VMR Building is missing :P

**Comment by Suchty112** [comment]
@LSS-Manager deploy

**Review by Suchty112 — APPROVED** [review]
LGTM . I do not know the price of the VMR building, so it is not correct.

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**